### ISSUE TITLE
[Bug]: SSH stopped working all of a sudden

### ISSUE THREAD

**Issue by mityukov** [beyondcode/tinkerwell-community#568]
### Operating system

macOS Moterey 12.3

### Laravel Version

8.83.1

### PHP Version

8.0.5

### Project Location

Remote (Server)

### Bug description

Hi, I have an SSH connection to a site on Laravel Forge and today it stopped working.

I click "Run" and the spinner just never end, whatever command I'm trying to execute.

Note: I can login into the site via iTerm just Ok, i.e., it doesn't look like an SSH key problem. Or, if it is — I'm not sure what to check next…

### Steps to reproduce

1. Have a site on Laravel Forge;
2. Add `~/.ssh/id_rsa.pub` into that site's "SSH Keys"
3. Log into this site using Terminal or iTerm — it's OK
4. Try something in tinker, while remotely on that server — it's OK
5. Setup a connection in Tinkerwell with `Auth method = Private Key` and the `Key = /home/user/.ssh/id_rsa`
6. Connect
7. Try running anything (like `echo "here"`)
8. it just starts spinning the cursor and it never ends…
9. Worked on Friday, nothing's changed besides updating of the macOS to 12.3…

<img width="467" alt="image" src="https://user-images.githubusercontent.com/1751910/159215345-fe0ef25d-fa85-4656-8ea6-66eb95ce5075.png">

**Comment by mpociot** [comment]
Hi!

Does the tab indicate that you are connected via SSH?
Can you try changing the "PHP binary path" to `echo` and re-run any command via SSH?
This should show you the command that is being executed via SSH.

**Comment by mityukov** [comment]
> Does the tab indicate that you are connected via SSH?

yes, I can see a banner "Connected to MYSITE" at the bottom and a color Circle to the left of the tab name at the top.

> Can you try changing the "PHP binary path" to echo and re-run any command via SSH?

The same result. No output — just the spinner that never stops…

**Comment by mityukov** [comment]
Just tried to login into forge again (in the Terminal), but with SSH key, defined explicitly: `ssh -i ~/.ssh/id_rsa forge@MYSITE` — still works. Could it be that Tinkerwell doesn't have access to id_rsa's content for some treason? How to check it?
